__tests__/app.test.js

```diff
@@ -35,4 +35,31 @@ describe('3. get /api/topics', () => {
             })
         })
     });
-});
\ No newline at end of file
+});
+
+describe('4. get /api/articles/:article_id', () => {
+    test('should return 200 and an object that matches the article_id for selected columns', () => {
+        return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({body}) => {
+            expect(body.articles).toEqual({
+                    author: "butter_bridge",
+                    title: "Living in the shadow of a great man",
+                    article_id: 1,
+                    body: "I find this existence challenging",
+                    topic: "mitch",
+                    created_at: "2020-07-09T20:11:00.000Z",
+                    votes: 100
+                })
+        })
+    });
+    test('should return 404 if article_id does not exist', () => {
+        return request(app)
+        .get("/api/articles/122")
+        .expect(404)
+        .then(({body}) => {
+            expect(body.message).toBe("no article_id found")
+        })
+    });
+})
```

Comment:
Great stuff so far but missing a 400 test here for if a the id is invalid, like if you tried /api/articles/banana